copyright

```diff
@@ -1860,6 +1860,12 @@ Copyright: Various
 License: CC0
 Comment: Taken from unsplash.com and cropped.
 
+Files:
+ outfit/old sword
+Copyright: RisingLeaf
+License: CC-BY-SA-4.0
+Comment: A sword modelled by RisingLeaf for a short mission line.
```

Comment:
Was the original source okay with you not mentioning them?

Reply:
removed it